packages/blockchain-link-types/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# 1.2.0
+
+-   npm-prerelease: @trezor/blockchain-link-types 1.1.1-beta.3 (2be3a59462)
+-   chore(suite): get rid of NETWORKS reexport (02bd51829a)
+-   feat(solana): validate send amount against rent (9f1f76b994)
+-   feat(staking): instant staking/unstaking success (6f42f9f86b)
+-   npm-prerelease: @trezor/blockchain-link-types 1.1.1-beta.2 (67e3933906)
```

Comment:
do we want these npm-prerelease lines here?

Reply:
I replied to it in comment above https://github.com/trezor/trezor-suite/pull/13923#discussion_r1726552536